docs/source/user_manual/modules_and_classes.rst

```diff
@@ -237,19 +283,48 @@ first in Chaco, and then moved into Enable.
   * :class:`~.ScatterInspector`
   * :class:`~.SelectableLegend`
 
+* :class:`~.SimpleInspectorTool`
 * :class:`enable.tools.traits_tool.TraitsTool`
+* :class:`~chaco.tools.traits_tool.TraitsTool`
```

Comment:
tangential question - these two look similar but im not sure what their differences are - and why